src/components/QueryBuilder.vue

```diff
@@ -63,6 +70,18 @@ export default Vue.extend( {
 			}
 
 			if ( !this.selectedItem || !this.textInputValue ) {
+				if ( !this.selectedItem ) {
```

Comment:
I'm not particularly happy about adding more business logic to this component. But even if we add it to an action now, we still have to touch it again when we implement adding more conditions, so we might as well keep it here until then.

Reply:
Indeed. My plan is to actually move everything into a js service like "Validator" or something which would accept a Form object and returns errors (I'm not sure about vue store action though but I can be convinced otherwise, as I'm not really expert in frontend) but this is all for future (and I will do it once the component becomes a bit stable, currently it's being changed a lot)